src/azure-cli/azure/cli/command_modules/vm/azure_stack/custom.py

```diff
@@ -0,0 +1,5644 @@
+              no_wait=False, authentication_type=None, admin_password=None, computer_name=None,
+              admin_username=None, ssh_dest_key_path=None, ssh_key_value=None, generate_ssh_keys=False,
+              availability_set=None, nics=None, nsg=None, nsg_rule=None, accelerated_networking=None,
+              private_ip_address=None, public_ip_address=None, public_ip_address_allocation='dynamic',
+              public_ip_address_dns_name=None, public_ip_sku=None, os_disk_name=None, os_type=None,
+              storage_account=None, os_caching=None, data_caching=None, storage_container_name=None, storage_sku=None,
+              use_unmanaged_disk=False, attach_os_disk=None, os_disk_size_gb=None, attach_data_disks=None,
+              data_disk_sizes_gb=None, disk_info=None,
+              vnet_name=None, vnet_address_prefix='10.0.0.0/16', subnet=None, subnet_address_prefix='10.0.0.0/24',
+              storage_profile=None, os_publisher=None, os_offer=None, os_sku=None, os_version=None,
+              storage_account_type=None, vnet_type=None, nsg_type=None, public_ip_address_type=None, nic_type=None,
+              validate=False, custom_data=None, secrets=None, plan_name=None, plan_product=None, plan_publisher=None,
+              plan_promotion_code=None, license_type=None, assign_identity=None, identity_scope=None,
+              identity_role=None, identity_role_id=None, application_security_groups=None, zone=None,
+              boot_diagnostics_storage=None, ultra_ssd_enabled=None,
+              ephemeral_os_disk=None, ephemeral_os_disk_placement=None,
+              proximity_placement_group=None, dedicated_host=None, dedicated_host_group=None, aux_subscriptions=None,
+              priority=None, max_price=None, eviction_policy=None, enable_agent=None, workspace=None, vmss=None,
+              os_disk_encryption_set=None, data_disk_encryption_sets=None, specialized=None,
+              encryption_at_host=None, enable_auto_update=None, patch_mode=None, ssh_key_name=None,
+              enable_hotpatching=None, platform_fault_domain=None, security_type=None, enable_secure_boot=None,
+              enable_vtpm=None, count=None, edge_zone=None, nic_delete_option=None, os_disk_delete_option=None,
+              data_disk_delete_option=None, user_data=None, capacity_reservation_group=None, enable_hibernation=None,
+              v_cpus_available=None, v_cpus_per_core=None, accept_term=None, disable_integrity_monitoring=False,
+              os_disk_security_encryption_type=None, os_disk_secure_vm_disk_encryption_set=None,
+              disk_controller_type=None):
+
+    from azure.cli.core.commands.client_factory import get_subscription_id
+    from azure.cli.core.util import random_string, hash_string
+    from azure.cli.core.commands.arm import ArmTemplateBuilder
+    from azure.cli.command_modules.vm.azure_stack._template_builder \
```

Comment:
I believe that we don't need to repeat a copy of the building ARM template part, which will make our code more complex and difficult to maintain
```suggestion
    from azure.cli.command_modules.vm._template_builder \
```